ci/test.sh

```diff
@@ -38,9 +38,6 @@ markEndOfBlock "Code Generator Testing"
 
 markStartOfBlock "Editmode Testing"
 
-rm -rf "${PROJECT_DIR}/workers/unity/Library/"
```

Comment:
IIRC, this looks like a holdover from the SDK scripts, which tested current/previous supported versions of Unity. Unity often falls over in mysterious ways if you change versions and don't clean up Temp/Library.

Reply:
For this script I had added it because I was running into it repeatedly even if the version did not change. It had side effects like it taking longer to reopen the project after tests were run, and have to reconfigure project layout. That problem seems to be gone now so I removed these lines to eliminate the side effects. I think overall the annoyance from the continuation of the side effects can still be considered to be worse than the risk of the Library folder causing mysterious failures. If the failures do happen at a significant frequency we can re-add these lines.